pycbc/waveform/generator.py

```diff
@@ -1198,57 +1199,57 @@ def generate(self, **kwargs):
 #
 
 
-def select_waveform_generator(approximant):
-    """Returns the single-IFO generator for the approximant.
-
-    Parameters
-    ----------
-    approximant : str
-        Name of waveform approximant. Valid names can be found using
-        ``pycbc.waveform`` methods.
-
-    Returns
-    -------
-    generator : (PyCBC generator instance)
-        A waveform generator object.
-
-    Examples
-    --------
-    Get a list of available approximants:
-    >>> from pycbc import waveform
-    >>> waveform.fd_approximants()
-    >>> waveform.td_approximants()
-    >>> from pycbc.waveform import ringdown
-    >>> ringdown.ringdown_fd_approximants.keys()
-
-    Get generator object:
-    >>> from pycbc.waveform.generator import select_waveform_generator
-    >>> select_waveform_generator(waveform.fd_approximants()[0])
-    """
-    # check if frequency-domain CBC waveform
+def select_waveform_generator(approximant, domain):
+    """Returns the single-IFO generator for the approximant."""
```

Comment:
Add some documentation here about what domain is, and what the accepted options are. Also, I'd set the default domain to None (`domain=None`), to keep the calling signature the same as it was in the past.